src/utils.js

```diff
@@ -90,6 +90,7 @@ function urlToHttpOptions (aUrl) {
  * @returns {number} Number of milliseconds to sleep until the next call to getEventsFromJournal
  */
 function parseRetryAfterHeader (header) {
+  logger.debug('Parsing Retry-After header')
```

Comment:
i meant debug log to log the header value, and the returned time in milli seconds so we know what the calculated time was after which the request would be retried again. Like: https://github.com/adobe/aio-lib-core-networking/blob/master/src/HttpExponentialBackoff.js#L125

Reply:
Oh I see, it makes more sense yeah good idea!